utils/git.go

```diff
@@ -267,3 +279,40 @@ func (m *manager) readRevisionFromPackedRef(ref string) {
 	m.err = errorutils.CheckError(errors.New("failed fetching revision from git config, from ref: " + ref))
 	return
 }
+
+func (m *manager) readMessage() {
+	if m.err != nil {
+		return
+	}
+	path := m.getPathHandleSubmodule()
+	r, err := git.PlainOpenWithOptions(path, &git.PlainOpenOptions{DetectDotGit: false})
+	err = errorutils.CheckError(err)
+	if err != nil {
+		m.err = err
+		return
+	}
```

Comment:
Can be consolidated to 1 line. Same comment for the 2 below `CheckError`s
```suggestion
	if errorutils.CheckError(err) != nil {
		m.err = err
		return
	}
```